modules/cluster-entitlements.adoc

```diff
@@ -17,33 +17,20 @@
 [id="cluster-entitlements_{context}"]
 = Internet and Telemetry access for {product-title}
 
-In {product-title} {product-version}, Telemetry is the component that provides
-metrics about cluster health and the success of updates. To perform subscription
-management, including legally entitling your purchase from Red Hat, you must use
-the Telemetry service and access the
-link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers] page.
+In {product-title} {product-version}, you require access to the internet to install and entitle your cluster. The Telemetry service, which runs by default to provide metrics about cluster health and the success of updates, also requires internet access. If your cluster is connected to the internet, Telemetry runs automatically, and your cluster is automatically entitled after installation.
```

Comment:
There is no automatic entitlement on OpenShift Cluster Manager, the cluster is simply registered and then the customer has to assign the correct entitlements.  So we should probably replace the end of this with..

> and your cluster is registered to OpenShift Cluster Manager, entitlements would then need to be allocated after installation.